### DISCUSSION TITLE
Launch flags/config file keys

### DISCUSSION THREAD

**Discussion by edwardjamesgaff** [coder/code-server#6049] [Q&A] [answered]
Is there a full list of launch flags/config file keys? [This](https://coder.com/docs/code-server/latest/FAQ#how-does-the-config-file-work) suggests using the --help flag but it doesn't seem to list all flags.

**Comment by edwardjamesgaff** [comment, marked as answer]
If anyone else is interested I believe the full list of flags is available by perusing the following:

https://github.com/coder/code-server/blob/main/src/node/cli.ts

**Reply by code-asher** [reply]
Yup anything without a description there is hidden by default from `--help` but maybe we should change that behavior.

**Reply by edwardjamesgaff** [reply]
It would be nice to have them documented in some way. I just started trying out code-server and I've been piecing together the different flags from random posts/the setup and configuration guide. Was just looking for a full list because I like being aware of all the configuration settings.

In the meantime, I'm happy and comfortable looking through the code so I'll mark that as the answer.
